__tests__/app.test.js

```diff
@@ -347,3 +422,37 @@ describe('GET api users', () => {
       });
   });
 });
+
+describe('GET api articles queries', () => {
+  describe(('Topic query'), () => {
+    test('GET:200 sends an array of users that have the queried topic', () => {
+    return request(app)
+      .get('/api/articles?topic=mitch')
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(Array.isArray(articles)).toBe(true);
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article.topic).toBe('mitch');
```

Comment:
You will want to check that all of the properties are here too (just in case your SQL hasn't worked) 🙂